### DISCUSSION TITLE
Job de Release falhando

### DISCUSSION THREAD

**Discussion by deividferreira** [wmixvideo/nfe#978] [Geral]
Desde a versão 4.0.41, me parece que o Job de release está falhando! Não sei como proceder com a correção... Estou abrindo a discussion para ciência de alguém que consiga resolvê-lo.

 Grato!

**Comment by fincatto** [comment]
Bom dia. 
O pessoal do OSSRH, que usamos para hospedar a lib, mudou a autenticação, quebrando nosso release. 
Resolvemos ontem com eles, embora ainda não esteja aparecendo no maven central. 
Estamos trabalhando nisso agora e cremos que, em breve, deve estar sincronizando novamente.

**Comment by fincatto** [comment]
Tá feito: https://central.sonatype.com/namespace/com.github.wmixvideo
